src/suvela/Exam3.java

```diff
@@ -0,0 +1,46 @@
+package suvela;
+
+public class Exam3 {
+	int count;
+
+	void duplicateElement(int[] arr) {
+		for (int i = 0; i < arr.length; i++) {
+			count = 0;
+			int no = arr[i];
+			for (int j = 0; j < arr.length; j++) {
+				if (no == arr[j]) {
+					count++;
+					arr[i] = 0;
+				}
+
+			}
+			if (count > 1 && arr[i] == 0)
```

Comment:
why we need arr[i] == 0 condition ? I think only count condition would be sufficient.

Reply:
ok, got it.